src/unixfs-data-format.md

```diff
@@ -0,0 +1,652 @@
+---
+title: UnixFS
+description: >
+  UnixFS is a Protocol Buffers-based format for describing files, directories,
+  and symlinks as dag-pb and raw DAGs in IPFS.
+date: 2025-03-01
+maturity: draft
+editors:
+  - name: David Dias
+    github: daviddias
+  - name: Jeromy Johnson
+    github: whyrusleeping
+  - name: Alex Potsides
+    github: achingbrain
+    affiliation:
+      name: Interplanetary Shipyard
+      url: https://ipshipyard.com/
+  - name: Peter Rabbitson
+    github: ribasushi
+  - name: Hugo Valtier
+    github: jorropo
+  - name: Marcin Rataj
+    github: lidel
+    affiliation:
+      name: Interplanetary Shipyard
+      url: https://ipshipyard.com/
+
+tags: ['data-formats']
+order: 1
+---
+
+# Node Types
+
+A :dfn[Node] is the smallest unit present in a graph, and it comes from graph
+theory. In UnixFS, there is a 1-to-1 mapping between nodes and blocks. Therefore,
+they are used interchangeably in this document.
+
+A node is addressed by a [CID]. In order to be able to read a node, its [CID] is
+required. A [CID] includes two important pieces of information:
+
+1. A [multicodec], simply known as a codec.
+2. A [multihash] used to specify the hashing algorithm, the hash parameters and
+   the hash digest.
+
+Thus, the block must be retrieved; that is, the bytes which ,when hashed using the
+hash function specified in the multihash, gives us the same multihash value back.
+
+In UnixFS, a node can be encoded using two different multicodecs, listed below. More details are provided in the following sections:
+
+- [`raw`](#raw-node) (`0x55`), which are single block files without any metadata.
+- [`dag-pb`](#dag-pb-node) (`0x70`), which can be of any other type.
+
+# `raw` Node
+
+The simplest nodes use `raw` encoding and are implicitly a :ref[File]. They can
+be recognized because their CIDs are encoded using the `raw` (`0x55`) codec:
+
+- The block is the file data. There is no protobuf envelope or metadata.
+- They never have any children nodes, and thus are also known as single block files.
+- Their size is the length of the block body (`Tsize` in parent is equal to `blocksize`).
+
+# `dag-pb` Node
+
+More complex nodes use the `dag-pb` (`0x70`) encoding. These nodes require two steps of
+decoding. The first step is to decode the outer container of the block. This is encoded using the IPLD [`dag-pb`][ipld-dag-pb] specification, which can be
+summarized as follows:
+
+```protobuf
+message PBLink {
+  // binary CID (with no multibase prefix) of the target object
+  optional bytes Hash = 1;
```

Comment:
It might be more helpful for first-timers if:
1.) there were a link to [the CID spec](https://github.com/multiformats/cid)
2.) it were more explicit that "no multibase prefix" here means actually zero bytes of prefix, as opposed to the "identity" (`0x00`) prefix used canonically in [CBOR CIDs](https://github.com/ipld/cid-cbor/?tab=readme-ov-file#syntax)

Reply:
Thank you, I've added basic link and explainer in d2c28a30c559508952fb44be10769d601a461b82

The "identity" prefix is specific to CBOR TAG 42 as noted in  https://github.com/ipld/cid-cbor/  
It is not used anywhere else so mentioning it here does not bring clarity, and adds unnecessary confusion. 

CBOR quirks should live in CBOR spec.
If we have a spec for dag-cbor other than https://ipld.io/specs/codecs/dag-cbor/ we can link to it from https://specs.ipfs.tech/ in the same section as UnixFS.